src/main/java/org/pac4j/springframework/web/CallbackController.java

```diff
@@ -50,7 +53,9 @@ public void callback(final HttpServletRequest request, final HttpServletResponse
         assertNotNull("config", config);
         final J2EContext context = new J2EContext(request, response, config.getSessionStore());
 
-        callbackLogic.perform(context, config, J2ENopHttpActionAdapter.INSTANCE, this.defaultUrl, this.multiProfile, this.renewSession);
+        callbackLogic.perform(context, config, J2ENopHttpActionAdapter.INSTANCE, this.defaultUrl,
+            this.saveInSession, this.multiProfile, this.renewSession,
+            config.getClients().getClients().get(0).getName());
```

Comment:
Not sure about this part.

Reply:
Nice try ;-) But in fact, this parameter is a default client (a specific property) you can define for this specific endpoint (especially requested by the JAX-RS implementation).
See: https://github.com/pac4j/j2e-pac4j/blob/master/src/main/java/org/pac4j/j2e/filter/CallbackFilter.java#L48